locales/en.json

```diff
@@ -1522,7 +1522,7 @@
     "everyThursday": "Every Thursday",
     "addToCal": "Add To Calendar",
     "createACalendarEvent": "Create A Calendar Event",
-    "kodadotWeeklyGenerativeDrop": "Kodadot weekly generative drop",
+    "KodaWeeklyGenerativeDrop": "Koda weekly generative drop",
```

Comment:
same here

Reply:
@hassnian Double checked everything, and looks good to me except one place, I kept **https://hello.kodadot.xyz/multi-chain/migrator** as it is, as i saw there is no valid web app up and running for **https://hello.koda.art/multi-chain/migrator**.